website/other/bezier-rs-demos/src/components/BezierDemoPane.ts

```diff
@@ -46,10 +46,10 @@ class BezierDemoPane extends HTMLElement implements DemoPane {
 	computeType!: ComputeType;
 
 	connectedCallback(): void {
-		this.id = `${Math.random()}`.substring(2);
 		this.computeType = "Parametric";
 
 		this.key = (this.getAttribute("name") || "") as BezierFeatureKey;
+		this.id = `bezier_${this.key}`;
```

Comment:
Is this unique across the three: linear, quadratic, cubic?

Reply:
Oh, this is `BezierDemoPane` not `BezierDemo`.

Reply:
Yup! The key is different for each (`intersectLinear`, `intersectQuadratic`, and `intersectCubic`).